staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -300,6 +314,55 @@ type VirtualMachineInstanceStatus struct {
 	// +listType=atomic
 	// +optional
 	MigratedVolumes []StorageMigratedVolumeInfo `json:"migratedVolumes,omitempty"`
+	// DeviceStatus reflects the state of devices requested in spec.domain.devices. This is an optional field available
+	// only when DRA feature gate is enabled
+	// +optional
+	DeviceStatus *DeviceStatus `json:"deviceStatus,omitempty"`
```

Comment:
nit
```suggestion
	DeviceStatuses *DeviceStatus `json:"deviceStatuses,omitempty"`
```

Or maybe just
```suggestion
	Devices *DeviceStatus `json:"devices,omitempty"`
```

Reply:
used DeviceStatuses, thats more in-line with PodStatus